InvoiceReader.Application/InvoiceReader.Application.csproj

```diff
@@ -1,12 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AutoMapper" Version="10.1.1" />
     <PackageReference Include="MediatR" Version="9.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="5.0.0" />
```

Comment:
What is the reason behind adding this reference?

Reply:
I had problem in building code without this package. I tried without installing it but didn't worked, that's why added.

Reply:
Okay, I understand the issue.

It has a dll reference. And this is the reason why it's working on my machine and not yours.
```
<ItemGroup>
    <Reference Include="Microsoft.Extensions.Caching.Abstractions">
      <HintPath>..\..\..\..\..\..\Program Files\dotnet\packs\Microsoft.AspNetCore.App.Ref\5.0.0\ref\net5.0\Microsoft.Extensions.Caching.Abstractions.dll</HintPath>
    </Reference>
  </ItemGroup>
  ```
Can you please remove your changes, I will create a separate issue regarding this. Fix that accordingly.

Reply:
Removed reference